### ISSUE TITLE
The watcher wrongly assumes that the field 'restart_when' is present in its instances' status file

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1078]
Originally reported of Google Code with ID 1022.

```
Then it can happen (for example after an upgrade) that watcher fails to read its status file.

```
Originally added on 2015-01-21 16:01:32 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed in

commit 4b59c9a007d1cae29199cfb192f04fd2e0bc1bec
Author: Petr Pudlak <pudlak@google.com>
Date:   Thu Jan 22 11:32:41 2015 +0100

    Handle missing 'restart_when' in watcher's state file
    
    Fixes issue #1022.
    
    Signed-off-by: Petr Pudlak <pudlak@google.com>
    Reviewed-by: Klaus Aehlig <aehlig@google.com>

```
Originally added on 2015-01-22 13:32:39 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:39:05 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:39:42 +0000 UTC.

Changed State: Released
